bindings/python/test_m68k.py

```diff
@@ -40,12 +40,14 @@
 	16: "Absolute Data Addressing  - Short",
 	17: "Absolute Data Addressing  - Long",
 	18: "Immidate value",
-}    
+}
 
 def print_insn_detail(insn):
     if len(insn.operands) > 0:
         print("\top_count: %u" % (len(insn.operands)))
 
+	print("\tgroups_count: %u" % len(insn.groups))
```

Comment:
@emoon : My bad, I apologize :( .... forgot to check in capstone/__init__.py  ; python binding have the magic trick implemented for all arch.

BTW : You should change the tabs to space in front of the print, or python will report : 
'''
  File "test_m68k.py", line 48
    print("\tgroups_count: %u" % len(insn.groups))
                                                 ^
TabError: inconsistent use of tabs and spaces in indentation
'''